chassis/core/src/main/java/com/griddynamics/jagger/rest/MasterRestController.java

```diff
@@ -0,0 +1,188 @@
+package com.griddynamics.jagger.rest;
+
+import com.griddynamics.jagger.dbapi.entity.TaskData;
+import com.griddynamics.jagger.master.CompositeTask;
+import com.griddynamics.jagger.master.SessionIdProvider;
+import com.griddynamics.jagger.master.TaskExecutionStatusProvider;
+import com.griddynamics.jagger.master.TaskIdProvider;
+import com.griddynamics.jagger.master.configuration.Configuration;
+import com.griddynamics.jagger.master.configuration.Task;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import com.google.common.collect.Lists;
+import com.google.common.collect.Maps;
+
+import java.util.List;
+import java.util.Map;
+
+import javax.annotation.Resource;
+
+/**
+ * Master node REST API controller based on Spring MVC.
```

Comment:
What do you think of adding more information to the javadoc in order to see what is the goal of the Class (Controller), what can it do, etc. ?
It is good practice that will simplify understanding the code in future.